core/block_svg.js

```diff
@@ -977,12 +1029,14 @@ Blockly.BlockSvg.prototype.onMouseMove_ = function(e) {
       // Switch to unrestricted dragging.
       Blockly.dragMode_ = Blockly.DRAG_FREE;
       Blockly.longStop_();
+      
+      // IS THIS COMMENT RELEVANT STILL???
```

Comment:
This is from one of my PRs to your local branch that I forgot to resolve. Would you mind removing it?

I believe the comment below is also irrelevant now too.  At least, as far as I can tell insertion markers/connection highlighting are working okay.  I remember having a conversation with Tim about this and we couldn't break it either... @rachel-fenichel: do you have any memory of what this means?